pkg/client/core/clienttester.go

```diff
@@ -24,11 +24,11 @@ type ClientV2Tester interface {
 //
 // The func is exposed as a var to allow tests to temporarily replace its
 // implementation, e.g. to return a fake.
-var NewClientV2Tester = func(t *testing.T, repo *api.Repo, emptyIndex bool, indexFile string) (ClientV2Tester, func(), error) {
+var NewClientV2Tester = func(t *testing.T, repo *api.Repo, emptyIndex bool, indexFile string) (ClientV2Tester, error) {
```

Comment:
```suggestion
var NewClientV2Tester = func(t *testing.T, repo *api.Repo, emptyIndex bool, indexFile string) (ClientV2Tester) {
```

please perform the rest of the required changes to use this signature

Reply:
ok!